x-pack/plugin/sql/qa/src/main/resources/datetime.csv-spec

```diff
@@ -121,6 +121,100 @@ SELECT WEEK(birth_date) week, birth_date FROM test_emp WHERE WEEK(birth_date) >
 2              |1953-01-07T00:00:00.000Z
 ;
 
+selectDateTruncWithDateTime
+schema::dt_hour:ts|dt_min:ts|dt_sec:ts|dt_millis:s|dt_micro:s|dt_nano:s
```

Comment:
Have to cast some columns to string as the `.XXX` is not properly read by the CSV infrastructure.

Reply:
Done: #46511